crates/rv/src/commands/ci.rs

```diff
@@ -49,10 +50,13 @@ struct CiInnerArgs {
     pub validate_checksums: bool,
     pub lockfile_path: Utf8PathBuf,
     pub install_path: Utf8PathBuf,
+    pub ruby_version: RubyRequest,
 }
 
 #[derive(Debug, thiserror::Error, miette::Diagnostic)]
 pub enum Error {
+    #[error("You must specify a ruby version to use")]
```

Comment:
```suggestion
    #[error("You must specify a Ruby version to use")]
```

Ruby should probably be capitalized here.